ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py

```diff
@@ -1271,3 +1271,59 @@ def test_tensorflow_linspace(
         axis=axis,
         on_device=on_device,
     )
+   
+
+# tile
+@st.composite
+def _multiple_shape_helper(draw):
+    input_dtype, input_array, input_shape = draw(
+        helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("numeric"),
+            ret_shape=True
+        ))
+    
+    # input = dtypes_and_input[1][0]
+    input_dims = len(input_shape)
+    
+    shape = draw(helpers.get_shape(
+        min_num_dims=1, max_num_dims=1,
+        min_dim_size=input_dims, max_dim_size=input_dims
+    ))
+    dt_n_multiples = draw(
+        helpers.dtype_and_values(
+            available_dtypes=["int32", "int64"],
+            shape=shape))  
+    # return dtypes_n_input, dt_n_multiples
+    return input_dtype, input_array, dt_n_multiples
+
+
+@handle_frontend_test(
+    fn_tree="tensorflow.tile",
+    all_arguments=_multiple_shape_helper()
+)
+def test_tensorflow_tile(
+    *,
+    all_arguments,
+    num_positional_args,
+    as_variable,
+    native_array,
+    frontend,
+    fn_tree,
+    on_device
+):
+    # dtypes_and_input, dt_and_multiples = all_arguments
+    input_dtype, input_matrix, dt_and_multiples = all_arguments
+    # real_dtype, input_matrix = dtypes_and_input
+    dt_mul, multiples = dt_and_multiples
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
```

Comment:
`input_dtypes` should get the dtypes of all the array arguments, so you need to add `dt_mul`.

Reply:
Then you are creating a list of lists. I think it should be `input_dtypes=input_dtype + dt_mul`.

Reply:
I think lists would have to be concatenated in some other way. Maybe I can try extending it like so:
input_dtype.extend(dt_mul)

Reply:
```AttributeError: 'UintDtype' object has no attribute 'extend'```

Reply:
what should I do in this case?

Reply:
Why did you decide to `extend`? Was there an error when using `+`?

Reply:
I think the error is caused because you are extending `input_dtype[0]` which is not a list. Try `input_dtype.extend(dt_mul)`

Reply:
@AnnaTz yes, something about ```input_dtype``` and ```dt_mul``` not being strings

Reply:
That's because again you had done `input_dtype[0] + dt_mul` instead of `input_dtype + dt_mul`.

Reply:
There's another index error: ```E   IndexError: list index out of range```
Should ```input_dtypes=input_dtype + dt_mul```  access the individual element of these lists?

Reply:
You should have a look at the log error to see where the error is happening. From what I see it's not because of the dtype list. I think you need to pass `num_arrays=2` to the `dtype_and_values` that is creating `mutliples.

Reply:
noted, I'm just getting the hang of interpreting error logs successfully 😅 
I've passed the argument, and am a little confused as to how it will be accessed later by the tile function

Reply:
```
ivy.exceptions.IvyBackendException: tensorflow: einops_repeat: Tensor is unhashable. Instead, use tensor.ref() as the key.

```
on StackOverflow, it says it might be due to version incompatibility. How can we circumnavigate this?

```
E   Falsifying example: test_tensorflow_tile(
E       all_arguments=(['int16'],
E        [array(0, dtype=int16)],
E        (['int32', 'int32'], [array([], dtype=int32), array([], dtype=int32)])),
E       num_positional_args=0,
E       native_array=[False],
E       as_variable=[False],
E       fn_tree='ivy.functional.frontends.tensorflow.tile',
E       frontend='tensorflow',
E       on_device='cpu',
E   )
```